pkgs/development/python-modules/minio/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchFromGitHub
+, urllib3, python-dateutil , pytz, faker, mock, nose }:
+
+buildPythonPackage rec {
+  pname = "minio";
+  version = "4.0.11";
+
+  src = fetchFromGitHub {
```

Comment:
Please add a comment on why you're not using `fetchPypi`.

Reply:
bumped to 4.0.13 and fetched from pypi.